pkg/apis/postgres-operator.crunchydata.com/v1/pgbackrest_types.go

```diff
@@ -0,0 +1,15 @@
+// Copyright 2021 - 2025 Crunchy Data Solutions, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package v1
+
+import (
+	"github.com/crunchydata/postgres-operator/pkg/apis/postgres-operator.crunchydata.com/v1beta1"
+)
+
+// PGBackRestArchive defines a pgBackRest archive configuration
+// +kubebuilder:validation:XValidation:rule=`!has(self.repoHost) || !has(self.repoHost.log) || !has(self.repoHost.log.path) || self.repoHost.volumes.additional.exists(x, self.repoHost.log.path.startsWith("/volumes/"+x.name))`,message=`log path is restricted to an existing additional volume`
```

Comment:
Right now I just have it set to validate that the `repoHost.log.path` isn't set or if it is, it's set to an additional volume configured in `repoHost`... Not sure if we want to allow other paths or not... Thoughts?

Reply:
I believe the repohost (as it now exists) will always have a `/tmp` directory and a `/pgbackrest/repoN` directory. Is that right and if so, would we want to allow logging to those? (Though actually: if we let logging into `/pgbackrest/repoN`, what subdirectories would we allow? Probably `/pgbackrest/repoN/logs/pgbackrest` only?)

Reply:
Oh, but `/pgbackrest/repoN/log` (or logs?) is our default, so... I suppose we could allow explicit setting of our default just to let people be explicit.

Reply:
Spoke with Chris about this. Going to hold off on allowing logging to `/tmp`